assets/src/edit-story/components/panels/shared/flipControls.js

```diff
@@ -31,30 +32,61 @@ import { ReactComponent as FlipHorizontal } from '../../../icons/flip_horizontal
 import { ReactComponent as FlipVertical } from '../../../icons/flip_vertical.svg';
 import Toggle from '../../form/toggle';
 
-function FlipControls({ value, onChange }) {
+const styledToggle = css`
+  width: 36px;
+  height: 36px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  float: left;
+`;
+
+const ToggleContainer = styled.div`
+  ${styledToggle}
+  margin-right: ${({ margin }) => (margin ? margin : '0px')};
+`;
+
+const ControlsContainer = styled.div`
+  width: fit-content;
+`;
+
+/**
+ * Get flip controls for flipping elements horizontally and vertically.
+ *
+ * @param {Object} value Element's flip object.
+ * @param {Object} onChange Callback to flip element.
+ * @param {number} margin Space between the two flip toggles (defaults to '8px').
+ * @return {*} Element or null if does not map to video/image.
```

Comment:
This component never really returns `null`.

Also, a docblock like this would be more accurate:

```js
/**
 * Displays flip controls for flipping elements horizontally and vertically.
 *
 * @param {Object} props Component props.
 * @param {Object} props.value Element's flip object.
 * @param {Object} props.onChange Callback to flip element.
 * @param {number} props.margin Space between the two flip toggles (defaults to '8px').
 * @return {*} Rendered component.
 */
```

Notice the `props.` prefix

Reply:
Done.